### PR TITLE
Minting

### PR DESCRIPTION
**Pull request by StephenButtolph**
Introduced fixed cap minting function to the P-chain.

### CODE REVIEW

File: genesis/genesis.go [hunk 1]
```diff
@@ -110,6 +113,7 @@ func FromConfig(config *Config) ([]byte, ids.ID, error) {
 				Amount:  json.Uint64(5 * units.MegaAvax),
 			},
 		)
+		initialSupply += 5 * units.MegaAvax
```
Comment (danlaine) [inline genesis/genesis.go:116]: Should we make these constants?

File: genesis/genesis.go [hunk 2]
```diff
@@ -129,6 +133,7 @@ func FromConfig(config *Config) ([]byte, ids.ID, error) {
 				RewardAddress: destAddr,
 			},
 		)
+		initialSupply += 20 * units.KiloAvax
```
Comment (danlaine) [inline genesis/genesis.go:136]: Same here: Should we make these constants?

File: vms/platformvm/reward.go [hunk 3]
```diff
@@ -4,22 +4,50 @@
...
+	"math/big"
 	"time"
 )
 
-// reward returns the amount of tokens to reward the staker with
-func reward(duration time.Duration, amount uint64, inflationRate float64) uint64 {
-	// TODO: Can't use floats here. Need to figure out how to do some integer
-	// approximations
+var (
+	maxSubMinConsumptionRate   = new(big.Int).SetUint64(MaxSubMinConsumptionRate)
```
Comment (danlaine) [inline vms/platformvm/reward.go:12]: Some comments here pls

Comment (StephenButtolph) [reply vms/platformvm/reward.go:12]: added a couple comments

File: genesis/genesis.go (hunk 1 above)
Comment (StephenButtolph) [reply genesis/genesis.go:116]: These values are going to get moved into the config to support the real genesis. So I'll address this in a future PR.

File: genesis/genesis.go (hunk 2 above)
Comment (StephenButtolph) [reply genesis/genesis.go:139]: Same comment as above

**Pull request merged**